src/AnnouncementBubble/NormalAnnouncementBubble.less

```diff
@@ -73,8 +73,9 @@ button.NormalAnnouncementBubble--menuIconContainer {
   .margin--top--m();
   .text--line-height-4();
   white-space: pre-wrap;
+  // TODO: Remove? Does Dewey work on IE?
```

Comment:
I think by itself it shouldn't but in conjunction with babel + the polyfils we use in most of our apps, I think it should work (but I may be wrong)